src/auto-evo/simulation/food_source/EnvironmentalFoodSource.cs

```diff
@@ -42,5 +44,55 @@ public override float TotalEnergyAvailable()
         {
             return totalEnvironmentalEnergySource;
         }
+
+        protected override float StorageScore(MicrobeSpecies species, Compound compound, Patch patch,
+            SimulationCache simulationCache)
+        {
+            if (simulationCache.DayNightCycleEnabled)
+            {
+                var dayNightCycleConfiguration = SimulationParameters.Instance.GetDayNightCycleConfiguration();
+                var nightTime = dayNightCycleConfiguration.DaytimePercentage * dayNightCycleConfiguration.HoursPerDay;
+
+                // If a species consumes a lot, it ought to store more.
+                // NB: this might artificially penalize overproducers but I'm willing to accept it for now - Maxonovien
```

Comment:
What does NB stand for?

Reply:
I would assume [nota bene](https://en.wikipedia.org/wiki/Nota_bene).

Reply:
Then this should be rewritten as `NOTE:`